src/test/java/robot/subsystems/drivetrain/SwerveDriveTest.java

```diff
@@ -0,0 +1,132 @@
+package robot.subsystems.drivetrain;
+
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import org.junit.*;
+
+import java.util.Arrays;
+
+import static frc.robot.Constants.SwerveDrive.*;
+
+public class SwerveDriveTest {

```

Comment:
Notice that 4 of your tests are failing probably due to the radius not being 1 meter